vc_assistance/indico_vc_assistance/templates/event_request_details.html

```diff
@@ -0,0 +1,82 @@
+{% extends 'events/requests/event_request_details.html' %}
+{% from 'message_box.html' import message_box %}
+
+{% set link_start_vc -%}
+    <a href="{{ url_for('vc.manage_vc_rooms', event) }}">
+{%- endset %}
+ {% set link_start_event_settings -%}
+    <a href="{{ url_for('event_management.settings', event) }}">
+{%- endset %}
+{% set link_end -%}
+    </a>
+{%- endset %}
+
+{% block content %}
+    {% if not user_authorized %}
+        {% call message_box('warning') %}
+            {% trans -%}
+                Unfortunately you are not allowed to request videoconference assistance.
+            {%- endtrans %}
+        {% endcall %}
+    {% elif not within_working_hours %}
+        {% call message_box('warning') %}
+            {% trans -%}
+                This event starts out of working hours, so it is not possible
+                to request assistance.
+                {{ link_start_event_settings }}Manage event settings{{ link_end }}.
+            {%- endtrans %}
+        {% endcall %}
+    {% else %}
+        {% if not request_accepted and not has_vc_rooms %}
+            {% call message_box('warning') %}
+                {% trans -%}
+                    This event doesn't have any videoconference room attached,
+                    so it is not possible to request assistance.
+                    {{ link_start_vc }}Manage videoconference rooms{{ link_end }}.
+                {%- endtrans %}
+            {% endcall %}
+        {% elif not request_accepted and not has_vc_capable_rooms %}
+            {% call message_box('warning') %}
+                {% trans -%}
+                    None of the physical rooms attached to this event have videoconference
+                    capabilities, so it is not possible to request assistance.
+                    {{ link_start_event_settings }}Manage event settings{{ link_end }}.
+                {%- endtrans %}
+            {% endcall %}
+        {% elif not requested_accepted and not has_vc_rooms_attached_to_capable %}
```

Comment:
@pferreir: Whoops, this typo caused the problem we had during the meeting :man_facepalming:, thanks!